src/Microsoft.AspNetCore.Http.Features/Authentication/SignInContext.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Microsoft.AspNetCore.Http.Features.Authentication
 {
+    [Obsolete("See https://go.microsoft.com/fwlink/?linkid=845470")]
```

Comment:
I thought you weren't going to check these Obsolete's in until after you got all of the other repos converted?